components/api_request_helper/api_request_helper_unittest.cc

```diff
@@ -54,13 +55,21 @@ class ApiRequestHelperUnitTest : public testing::Test {
 
   void SetInterceptor(const std::string& expected_method,
                       const GURL& expected_url,
-                      const std::string& content_to_respond) {
+                      const std::string& content_to_respond,
+                      bool enable_cache) {
     url_loader_factory_.SetInterceptor(base::BindLambdaForTesting(
         [&, expected_method, expected_url,
          content_to_respond](const network::ResourceRequest& request) {
           url_loader_factory_.ClearResponses();
           EXPECT_EQ(request.url, expected_url);
           EXPECT_EQ(request.method, expected_method);
+          if (enable_cache) {
+            EXPECT_EQ(request.load_flags, net::LOAD_DO_NOT_SAVE_COOKIES);
+          } else {
+            EXPECT_EQ(request.load_flags, net::LOAD_DO_NOT_SAVE_COOKIES |
+                                              net::LOAD_BYPASS_CACHE |
+                                              net::LOAD_DISABLE_CACHE);
+          }
```

Comment:
Can we have some test case that actually sets enable_cache to true?
As far as I can see, we only executes the enable_cache = false path in all tests here.